app/components/dex-create.jsx

```diff
@@ -71,13 +80,28 @@ export class DexCreate extends Component {
             </div>
             <div className="form-group">
               <label htmlFor="generation">Generation</label>
-              <select className="form-control" ref={(c) => this._generation = c} defaultValue="7">
+              <select className="form-control" onChange={this.handleChange} value={this.state.generation}>
```

Comment:
since we destructure `generation` on line 67, we can just use `generation` instead of `this.state.generation`